packages/salesforcedx-vscode-apex-debugger/tsconfig.json

```diff
@@ -15,9 +12,5 @@
     "preserveConstEnums": true,
     "strict": true
   },
-  "exclude": [
-    "node_modules",
-    ".vscode-test",
-    "out"
-  ]
-}
\ No newline at end of file
+  "exclude": ["node_modules", ".vscode-test", "out"]
+}
```

Comment:
@vazexqi Do you have an idea why VS Code keeps formatting the JSON files like this?